README.md

```diff
@@ -227,7 +204,64 @@ transcript = await chat_exporter.export(channel, guild=channel.guild, asset_hand
 
 
 ---
-## Additional Functions
+## Additional Functions/Features
+
+<details><summary><b>Asset handler</b></summary>
+In order to prevent the transcripts from being broken either when a channel is deleted or by the newly introduced 
+restrictions to media links in discord, chat-exporter now supports an asset handler. 
+
+`chat_exporter.AttachmentHandler` servers as a template for you to implement your own asset handler. 
+As example we provide two basic versions of an asset handler, one that stores the assets locally and one that 
+uploads them to a discord. 
+Of course the second one is also in some sense broken, but it should give a good idea on how to implement such an 
+`AttachmentHandler`.
+
+If you don't specify an asset handler, chat-exporter will use the normal (proxy) urls for the assets.
+The important part of your implementation is, that you have to overwrite the url and proxy_url attribute of the 
+Attachment in your implementation of `AttachmentHandler`. The url attribute should be the url where the asset is available.
+
+
+**Example:**
+
+Assuming you have a file server running, which serves the content of the folder `/usr/share/assets/` 
```

Comment:
Do we need an example for `AttachmentToDiscordChannelHandler` too? May be good.

Also can put these both behind `<detail>`